examples/livekit/README.md

```diff
@@ -0,0 +1,157 @@
+# Using Livekit with STUNner
+
+This document guides you through the [LiveKit](https://livekit.io/) installation into Kubernetes, when it is used together with the STUNner WebRTC gateway.
+
+In this demo you will learn the following steps:
+- integrate a typical WebRTC application server with STUNner
+- deploy the LiveKit server into Kubernetes
+- configure STUNner to expose LiveKit to clients
+
+## Intallation
+
+## Prerequisites
+
+The below installation instructions require an operational cluster running a supported version of Kubernetes (>1.22). You can use any supported platform, any hosted or private Kubernetes cluster, but make sure that the cluster comes with a functional load-balancer integration (all major hosted Kubernetes services should support this). Otherwise, STUNner will not be able to allocate a public IP address for clients to reach your WebRTC infra.
+
+Unfortunately, Minikube is not supported for this demo. The reason is that [Let's Encrypt certificate is not available with nip.io] (https://medium.com/@EmiiKhaos/there-is-no-possibility-that-you-can-get-lets-encrypt-certificate-with-nip-io-7483663e0c1b). Later on you will learn more about this certificate.
+
+## Setup
+
+The figure below shows how LiveKit is deployed into Kubernetes without the contstraints of running in host network (`hostNetwork: true`). In this setup LiveKit uses STUNner as a 'local' `STUN` and `TURN` server which enhances the security and saves the overhead cost of using public a `STUN` server. 
+In this tutorial we deploy a video room example using [LiveKit's React SDK](https://github.com/livekit/livekit-react/tree/master/example), [LiveKit server](https://github.com/livekit/livekit) for media exchange, an `Ingress gateway` to provide secure connections, and configure STUNner to expose the LiveKit server pool to clients. 
+
+![STUNner LiveKit intergration deployment architecture](../../doc/stunner_livekit.svg)
+
+## Ingress
+
+In order to make this example work you must have an ingress controller installed in your cluster.
+
+```
+helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx
+helm repo update
+helm install ingress-nginx ingress-nginx/ingress-nginx
+```
+
+## Installation guide
+
+Let's start with a disclaimer. The LiveKit client example(browser) must have secure HTTP connection in order to work because, [getUserMedia](https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices/getUserMedia#browser_compatibility) is available only in secure contexts. This induces that the client-server(LiveKit-server) connection must be secure too. According to the [docs](https://docs.livekit.io/deploy/#domain,-ssl-certificates,-and-load-balancer) and our experiences, self-signed certs do not work.
```

Comment:
HTTP -> HTTPS
Self-signed certs works for the getUserMedia, etc... but will not work with secure websocket. So, we should mention that.